File: src/lib/services/clipboard.ts
/**
 * 剪贴板服务
 * @deprecated 请使用 api.clipboard 代替
 */

import { api } from '$lib/api';

export const readClipboard = () => api.clipboard.read();
export const writeClipboard = (text: string) => api.clipboard.write(text);

File: src-tauri/src/commands/clipboard.rs
//! 剪贴板命令

use crate::services::ClipboardService;

#[tauri::command]
pub fn read_clipboard() -> Result<String, String> {
    ClipboardService::read()
}

#[tauri::command]
pub fn write_clipboard(text: String) -> Result<(), String> {
    ClipboardService::write(text)
}
